test/e2e/autotls/auto_tls_test.go

```diff
@@ -57,6 +59,32 @@ func TestTls(t *testing.T) {
 	t.Run(env.AutoTLSTestName, testAutoTLS)
 }
 
+func TestTlsDisabledWithAnnotation(t *testing.T) {
+	clients := e2e.SetupWithNamespace(t, test.TLSNamespace)
+
+	names := test.ResourceNames{
+		Service: test.ObjectNameForTest(t),
+		Image:   "runtime",
+	}
+	test.EnsureTearDown(t, clients, &names)
+
+	objects, err := v1test.CreateServiceReady(t, clients, &names, rtesting.WithServiceAnnotations(map[string]string{networking.DisableAutoTLSAnnotationKey: "true"}))
+	if err != nil {
+		t.Fatalf("Failed to create initial Service: %v: %v", names.Service, err)
+	}
+
```

Comment:
Can we compose the checks into one wait?

Reply:
I followed the model of the other test; personally I feel like the separate checks can help with debugging the test